src/editors/containers/VideoUploadEditor/hooks.js

```diff
@@ -52,30 +18,21 @@ export const onVideoUpload = () => {
   return module.postUploadRedirect(storeState);
 };
 
-const getFileExtension = (filename) => filename.slice(Math.abs(filename.lastIndexOf('.') - 1) + 2);
-
-export const fileValidator = (setLoading, setErrorMessage, uploadVideo) => (file) => {
-  const supportedFormats = Object.keys(extToMime);
-  const ext = getFileExtension(file.name);
-  const type = extToMime[ext] || '';
-  const newFile = new File([file], file.name, { type });
-
-  if (supportedFormats.includes(ext)) {
-    uploadVideo({
-      supportedFiles: [newFile],
-      setLoadSpinner: setLoading,
-      postUploadRedirect: onVideoUpload(),
-    });
-  } else {
-    const errorMsg = 'Video must be an MP4 or MOV file';
-    setErrorMessage(errorMsg);
-  }
+export const useUploadVideo = async ({
```

Comment:
This is not a react hook, and so should not be named starting with use. 
Additionally, its not adding much so perhaps the dispatch call can be moved directly into the component?
Is the idea to wrap it here to make it easier to mock?